edgedb/_testbase.py

```diff
@@ -85,6 +85,7 @@ def _start_cluster(*, cleanup_atexit=True):
             f"--emit-server-status={status_file_unix}",
             "--port=auto",
             "--auto-shutdown",
+            "--generate-self-signed-cert",
```

Comment:
This wouldn't work with older servers, we need to add a version check here, or react to unknown option error

Reply:
Ah right, let me fix that.

Reply:
Fixed - I think an option check in `--help` is simpler (maybe more reliable?) than a version check?